src/utils/polyfills.js

```diff
@@ -242,12 +237,12 @@ if (!''.trim)
         // Should work in IE8 because Element.prototype instanceof Node is true according to the specs
```

Comment:
Not sure if we still need this as internet explorer is totally deprecated but it's probably not the purpose of this PR :)

Reply:
Noted it for later ! :D